packages/client/components/SpotlightModal.tsx

```diff
@@ -0,0 +1,208 @@
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center'
+})
+
+const ReflectionWrapper = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  position: 'absolute',
+  top: `calc(${SELECTED_HEIGHT_PERC / 2}% - ${ElementHeight.REFLECTION_CARD / 2}px)`,
+  left: `calc(50% - ${ElementWidth.REFLECTION_CARD / 2}px)`,
+  zIndex: ZIndex.REFLECTION_IN_FLIGHT_LOCAL
+})
+
+const SearchInput = styled('input')({
+  appearance: 'none',
+  border: `1px solid ${PALETTE.SKY_500}`,
+  borderRadius: 4,
+  boxShadow: `0 0 1px 1px ${PALETTE.SKY_300}`,
+  color: PALETTE.SLATE_700,
+  display: 'block',
+  fontSize: 14,
+  lineHeight: '24px',
+  outline: 'none',
+  padding: '6px 0 6px 39px',
+  width: '100%',
+  '::placeholder': {
+    color: PALETTE.SLATE_600
+  }
+})
+
+const SearchItem = styled(MenuItemLabel)({
+  overflow: 'visible',
+  padding: 0,
+  position: 'absolute',
+  bottom: -16,
+  width: ElementWidth.REFLECTION_CARD
+})
+
+const StyledMenuItemIcon = styled(MenuItemComponentAvatar)({
+  position: 'absolute',
+  left: 8,
+  top: 8
+})
+
+const SearchIcon = styled(Icon)({
+  color: PALETTE.SLATE_600,
+  fontSize: ICON_SIZE.MD24
+})
+
+const StyledCloseButton = styled(PlainButton)({
+  height: 24,
+  position: 'absolute',
+  right: 16
+})
+
+const CloseIcon = styled(Icon)({
+  color: PALETTE.SLATE_600,
+  cursor: 'pointer',
+  fontSize: ICON_SIZE.MD24,
+  '&:hover,:focus': {
+    color: PALETTE.SLATE_800
+  }
+})
+
+interface Props {
+  closeSpotlight: () => void
+  meeting: SpotlightModal_meeting
+  flipRef: (instance: HTMLDivElement) => void
+}
+
+const SpotlightModal = (props: Props) => {
+  const {closeSpotlight, meeting, flipRef} = props
+  const {spotlightReflection} = meeting
+  const isDesktop = useBreakpoint(Breakpoint.NEW_MEETING_SELECTOR)
+  const handleKeyDown = (e: KeyboardEvent<HTMLDivElement>) => {
+    if (e.key === 'Escape') closeSpotlight()
+  }
+  const reflectionGroupsCount = 0
+  if (!spotlightReflection) return null
+  return (
+    <>
+      <ModalContainer onKeyDown={handleKeyDown} isDesktop={isDesktop}>
+        <SelectedReflectionSection>
+          <TopRow>
+            <Title>Find cards with similar reflections</Title>
+            <StyledCloseButton onClick={closeSpotlight}>
+              <CloseIcon>close</CloseIcon>
+            </StyledCloseButton>
+          </TopRow>
+          <SearchItem>
+            <StyledMenuItemIcon>
+              <SearchIcon>search</SearchIcon>
+            </StyledMenuItemIcon>
+            <SearchInput
+              autoFocus
+              autoComplete='off'
+              name='search'
+              placeholder='Or search for keywords...'
+              type='text'
+            />
+          </SearchItem>
+        </SelectedReflectionSection>
+        <SimilarReflectionGroups>
+          {reflectionGroupsCount === 0 ? <SpotlightEmptyState /> : null}
+        </SimilarReflectionGroups>
+      </ModalContainer>
+      <ReflectionWrapper ref={flipRef}>
```

Comment:
The modal needs to fade in after the reflection animation starts doing its thing so I'm absolutely positioning it outside of the modal